docs/_includes/themes/zeppelin/_navigation.html

```diff
@@ -53,6 +53,7 @@
                 <li><a href="{{BASE_PATH}}/interpreter/jdbc.html">JDBC</a></li>
                 <li><a href="{{BASE_PATH}}/interpreter/lens.html">Lens</a></li>
                 <li><a href="{{BASE_PATH}}/interpreter/markdown.html">Markdown</a></li>
+                <li><a href="{{BASE_PATH}}/interpreter/python.md">Python</a></li>
```

Comment:
Hi @hriviere :) 
You need to change `interpreter/python.md` -> `interpreter/python.html` like others.